### DISCUSSION TITLE
specify different column names in SQLModel to match different naming convention

### DISCUSSION THREAD

**Discussion by matthiasburger** [fastapi/sqlmodel#576] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the SQLModel documentation, with the integrated search.
- [X] I already searched in Google "How to X in SQLModel" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to SQLModel but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to SQLModel but to [SQLAlchemy](https://github.com/sqlalchemy/sqlalchemy).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from typing import Optional
from uuid import UUID

from sqlmodel import SQLModel, Field


class ProjectBase(SQLModel):
    """
    Entity for the inspections.
    """
    guid: Optional[UUID] = Field(default=None, primary_key=True)
    manager_id: Optional[UUID] = Field(default=None)
```

### Description

Our SQL-naming conventions are, that the column-names are written in camelCase. So we should have `managerId` instead of `manager_id`. But in python we want to use the correct python naming convention in snake_case. 
I tried to use the event with column_reflect from SQLAlchemy, but it doesn't work - and doesn't change my column names.

edit: I don't want to change each single column, but more change the columns names dynamically with a function. Like the `def __tablename__(cls):` thingy for table-names.

### Operating System

Windows, macOS

### SQLModel Version

0.0.8

### Python Version

3.10.10

**Comment by antont** [comment]
We are using SA Column for this. Had an `avatar` field earlier, and it's still in the DB but not used in newer data, so the attr name in the class was changed, there's now a property for it instead.

```
    avatar_: Optional[str] = Field(sa_column=Column("avatar", String, default=None))
```

So you would do:

```
    manager_id: UUID = Field(sa_column=Column("managerId", String)) #or so
```

**Reply by matthiasburger** [reply]
ehm, yeah.. that's the easy answer - I think I was too fast with my question.
What I mean is, how to change the column names in general, dynamically - not specifying every single column-name.

**Reply by rivamarco** [reply]
Anything new on this?
We also need something like this for having uppercase columns.

Thanks!
